README.md

```diff
@@ -1,3 +1,3 @@
 ## Kafka App
```

Comment:
Why is this Title defined as a second level header? I mean, why not start with the first level?

```md
# Kafka App
```